CombineUnloadAIDriver.lua

```diff
@@ -2068,6 +2078,11 @@ function CombineUnloadAIDriver:onBlockingOtherVehicle(blockedVehicle)
 	end
 end
 
+--TODO: create an Int Setting for combinesListHUDOffset
```

Comment:
Actually, the entire combine list should be encapsulated by a single object. But that's for later.

Reply:
okay then I will won't think about that one for now